packages/core/src/utils/ValidJWEStucture.ts

```diff
@@ -0,0 +1,5 @@
+import type { EncryptedMessage } from '../types'
+
+export function validJWEStucture(message: any): message is EncryptedMessage {
+  return message.protected && message.iv && message.ciphertext && message.tag
```

Comment:
As the type of message is any, we can't be certain it is an object.

```suggestion
  return message && typeof message === 'object' && message.protected && message.iv && message.ciphertext && message.tag
```